.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bug: '
+labels: bug
+assignees: ''
+
+---
+
+**Note: For support questions, please use the [Stoplight Community forum](https://community.stoplight.io)**. This repository's issues are reserved for feature requests and bug reports. If you are unsure if you are experiencing a bug, the [Community forum](https://community.stoplight.io) is a great place to start.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. When I run this command: 'prism mock http://gist.github.com/an-esxample-spec-file.yaml --foo --bar`
+2. Then make the following HTTP request: `curl GET ...`
+3. Then something unexpected happens...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. macOS 10.14.4]
+ - Prism Version [e.g. v3.0.0]
+ - Install method [e.g. binary, npm, yarn, docker]
+
+**Additional context**
```

Comment:
I'd really remove this — if people need to put additional details, they'll put them anyway, regardless of whether we'll write this or not.

Reply:
Agreed.